### ISSUE TITLE
Blackfoot River Collection

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/riverscapes-tools#867]
I have refactored to the project merge tool to work off of a collection (instead of tags). 

Please add all the necessary HUC10 projects to the [Blackfoot River Collection](https://data.riverscapes.net/c/223c7eb2-6c39-437f-8421-428e113126ae/) so that I can run the merge tool for @AldenShallcross for each project type. VBET is already done.

Once the collection has all the necessary projects for a certain type, please check the box and write a one line comment below ("VBET ready" so that I get a notification). And once the merge is complete I will update the model names below to the merged project (see VBET).

- [x] [RSC](https://data.riverscapes.net/p/aac5bbe0-7af2-467c-b0d8-55ea050756ac/)
- [x] [BRAT](https://data.riverscapes.net/p/9ac083d2-9560-4930-a3f2-1de6f4baca0c/)
- [x] [Anthro](https://data.riverscapes.net/p/2cad57f6-655d-4252-adcc-206da2706e62/)
- [X] [VBET](https://data.riverscapes.net/rv/b7cfb874-e21d-4952-833a-e5b1ff176aac)
- [x] [RCAT](https://data.riverscapes.net/p/59d13ee2-877a-4915-ac26-8da2ed4d24bc)
- [x] [RME](https://data.riverscapes.net/p/84d8f0c5-6415-4d4f-abd3-f9099877673d/)

# Important Note!
Don't add the merged projects to the collection! The merged projects are the same type as the input HUC10 projects. The merge tool will see them as inputs on subsequent re-runs of the tool. Snake eating its tail...

FYI @joewheaton

**Comment by jtgilbert** [comment]
@philipbaileynar I think I need permissions.

**Comment by philipbaileynar** [comment]
invite sent

**Comment by jtgilbert** [comment]
All these should be done.

**Comment by philipbaileynar** [comment]
These are all merged and posted above.
